docs/admin.md

```diff
@@ -0,0 +1,88 @@
+# Administrator Guide
+
+## Backstage Operator configuration
+
+### Context
+
+As it is described in Design doc (TODO), Backstage CR's desired state is defined using layered configuration approach, which means:
+- By default each newly created Backstage CR uses Operator scope Default Configuration
+- Which can be fully or partially overriden for particular CR instance using ConfigMap with witn name pointed in BackstageCR.spec.RawConfig 
+- Which in turn can be customized by other BackstageCR.spec fields (see Backstage API doc)
+
+Cluster Administrator may want to customize Default Configuration due to internal preferences/limitations, for example:
+- Preferences/restrictions for Backstage and|or PostgreSQL images due to  Airgapped environment.
+- Existed Taints and tolerations policy, so Backstage Pods have to be configured with certain tolerations restrictions.
+- ...
+
+Default Configuration is implemented as a ConfigMap called *backstage-default-config*, deployed on *backstage-system* namespace and mounted to Backstage controller container as a */default-config* directory.
+This config map contains the set of keys/values which maps to file names/contents in the */default-config*.
+These files contain yaml manifests of objects used by Backstage controller as an initial desired state of Backstage CR according to Backstage Operator data model.
+(TODO: link to the diagram here) 
+
+Mapping of configMap keys (yaml files) to runtime objects (NOTE: for the time (Dec 20'23) it is a subject of change):
+| Key/File name         | k8s/OCP Kind        | Mandatory*    |Notes                                      |
+| ----------------------|:-------------------:| --------------|------------------------------------------:|
+| deployment.yaml       | appsv1.Deployment   | Yes           | Backstage deployment |
+| service.yaml          | corev1.Service      | Yes           | Backstage Service |
+| db-statefulset.yaml   | appsv1.Statefulset  | For DB enabled| PostgreSQL statefulSet    |    
+| db-service.yaml       | corev1.Service      | For DB enabled| PostgreSQL Service   |
+| db-secret.yaml        | corev1.Secret       | For DB enabled| Secret to connect Backstage to PSQL   |
+| route.yaml            | openshift.Route     | No (for OCP)  | Route exposing Backstage service    |
+| app-config.yaml       | corev1.ConfigMap    | No            | Backstage app-config.yaml    |
```

Comment:
But to me (even if it is subject to changes), this should strive to reflect the current state, no? And it would need to be updated accordingly in #70..
Because that's what the Docs team will leverage to begin with.

Reply:
Yes, the goal is to reflect #70

Reply:
Can we make the two PRs (#64 and #70) independent?
The focus right now is to allow QE and docs teams to get started as soon as possible with what we have already, and a minimum of documentation is needed by the docs team (and soon other users).
So to me, this whole table could be removed from this PR and moved to #70 instead. WDYT?

This will also allow us to merge #64 as soon as possible and fix https://github.com/janus-idp/operator/issues/108

Reply:
Well, I would not like us support several versions of documents at this stage (before GA)
IMO it is quite labor intensive for near zero profit.

But, OK, I added one more "ver" field to give an idea (HTH temporarily)

Reply:
Alright, sounds good then. Thanks.